hack/e2e-suite/monitoring.sh

```diff
@@ -72,8 +72,17 @@ function influx-data-exists {
   local retry_delay=30 #seconds
   local influx_ip=$("${KUBECTL}" get -o json pods influx-grafana | grep hostIP | awk '{print $2}' | sed 's/["|,]//g')
   local influx_url="http://$influx_ip:8086/db/k8s/series?u=root&p=root"
-  if ! curl --retry $max_retries --retry-delay $retry_delay -G $influx_url --data-urlencode "q=select * from stats limit 1" \
-    || ! curl --retry $max_retries --retry-delay $retry_delay -G $influx_url --data-urlencode "q=select * from machine limit 1"; then
+  local ok="false"
+  for i in `seq 1 10`; do
+    if curl --retry $max_retries --retry-delay $retry_delay -G $influx_url --data-urlencode "q=select * from stats limit 1" \
```

Comment:
Honestly, I'm not sure how this works without auth.  But perhaps I'm confused here.